prep-javascript-dom-manipulation/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript DOM Manipulation</title>
+  <style>
+    #container {
+      border: 2px solid green;
+    }
+  </style>
+</head>
+
+<body>
+  <div id="container">
+    <p class="first-paragraph">
+      I really like coding.
+    </p>
+    <p class="second-paragraph">
+      I also really like to watch television and eating pancakes.
+    </p>
+    <p class="third-paragraph">
+      I hope one day to meet people that also like coding, television, and pancakes.
+    </p>
+  </div>
+  <script>
+    document.querySelector('p.second-paragraph') = $secondParagraph
+    console.log($secondParagraph)
+    document.querySelector('p.third-paragraph') = $thirdParagraph
```

Comment:
Hi Nadia, for this exercise we don't want to edit the HTML code, but instead use JavaScript to update the DOM after the page is actually loaded.